projects/Reth_contributor.md

```diff
@@ -0,0 +1,104 @@
+# Project Proposal
+
+Become a core contributor for [Reth](https://github.com/paradigmxyz/reth), a new Ethereum execution client written in Rust!
+
+## Motivation
+
+Client diversity is one of the most important features of Ethereum as it helps to further decentralise the network. Right now there are already several implementations for the Ethereum protocol, divided in consensus and execution clients, as you can see from the image below.
+
+![client_diversity.png](Reth_contributor/client_diversity.png)
+
+Reth is an execution client and it’s the first one written in the Rust language, another good thing since it creates diversity also on the programming languages used for Ethereum clients. It aims to be very fast (Rust language is known for its speed and reliability) and optimised in order to be able to satisfy different users:
+
+- big companies who need power and speed, have lots of money to invest in machines and infrastructures.
+- normal users who just want to run their own node with a normal computer.
+
+I am really excited to contribute to such a significant software project. Working on it lets me study and gain deep knowledge of various aspects of the Ethereum protocol.
+
+## Project description
+
+My project for this EPF is divided into two parts:
+
+1. Create an implementation of [EIP-7212](https://eips.ethereum.org/EIPS/eip-7212) for Reth. This EIP aims to introduce a precompile for elliptic curve operations on the ”*sepc256r1*” curve, which is different from the ”*sepc256k1*” used on Ethereum (and Bitcoin) for signatures verification.
+2. Solve a lot of smaller issues on the Reth repo, covering various aspects of the protocol and of an Ethereum node.
+
+## Specification
+
+### EIP-7212
+
+As of `FORK_TIMESTAMP` in the integrated EVM chain, add precompiled contract `P256VERIFY` for signature verifications in the “*secp256r1*” elliptic curve at address `PRECOMPILED_ADDRESS` in `0x19`.
+
+// **TODO**: I have to study further this EIP and start planning a more concrete specification and documentation of it.
+
+### Smaller Issues
+
+Every smaller issue has its own mini-proposal within the overall project proposal outlining the problem statement, proposed solution, and challenges.
+
+// THIS IS A WORK-IN-PROGRESS LIST
```

Comment:
Interesting what you did here with all smaller tasks! I wonder if it would be easier to avoid using a directory and just add all details here in spoiler tags, markdown `<details>  <summary>` feature. Or maybe link corresponding issue/PR with specs and tracking

Reply:
Yes, I think it's better to have all in one page and use a toogle list for every small issue so that it's easy to update and mantain it on the repo.